src/SFA.DAS.Assessor.Functions/Functions/Ofqual/OfqualImportFunction.cs

```diff
@@ -0,0 +1,78 @@
+        {
+            try
+            {
+                if (myTimer.IsPastDue)
+                {
+                    log.LogInformation("OfqualImport timer trigger has started later than scheduled");
+                }
+
+                // Function input comes from the request content.
+                string instanceId = await starter.StartNewAsync(nameof(RunOfqualImportOrchestrator), null);
+
+                log.LogInformation("Started OfqualImport with ID = '{instanceId}'.", instanceId);
+            }
+            catch (Exception ex)
+            {
+                log.LogError(ex, $"OfqualImport failed. Exception message: {ex.Message}");
+                throw;
+            }
+        }
+
+        [FunctionName(nameof(RunOfqualImportOrchestrator))]
+        public async Task<int> RunOfqualImportOrchestrator([OrchestrationTrigger] IDurableOrchestrationContext context, ILogger log)
+        {
+            ILogger replaySafeLogger = context.CreateReplaySafeLogger(log);
+
+            var parallelTasks = new List<Task<int>>
+            {
+                DoOrganisationsTasks(context),
+                DoQualificationsTasks(context)
+            };
+
+            await Task.WhenAll(parallelTasks);
+
+            replaySafeLogger.LogInformation("Loading Ofqual Standards for Ofqual Organisations using data in staging tables.");
+
+            int standardsLoaded = await context.CallActivityAsync<int>(nameof(OfqualStandardsLoader.LoadStandards), null);
+
+            replaySafeLogger.LogInformation("Ofqual import complete. {recordsInserted} Ofqual standards were loaded for Ofqual organisations.", standardsLoaded);
+
+            return standardsLoaded;
+        }
+
+        private async Task<int> DoOrganisationsTasks(IDurableOrchestrationContext context)
+        {
+            string organisationsDataFilePath = await context.CallActivityAsync<string>(nameof(OrganisationsDownloader.DownloadOrganisationsData), null);
+            var ofqualOrganisationData = await context.CallActivityAsync<IEnumerable<OfqualOrganisation>>(nameof(OfqualDataReader.ReadOrganisationsData), organisationsDataFilePath);
+            int recordsStaged = await context.CallActivityAsync<int>(nameof(OrganisationsStager.InsertOrganisationsDataIntoStaging), ofqualOrganisationData);
+
+            return recordsStaged;
+        }
+
+        private async Task<int> DoQualificationsTasks(IDurableOrchestrationContext context)
+        {
+            string qualificationsDataFilePath = await context.CallActivityAsync<string>(nameof(QualificationsDownloader.DownloadQualificationsData), null);
+            var ofqualQualificationData = await context.CallActivityAsync<IEnumerable<OfqualStandard>>(nameof(OfqualDataReader.ReadQualificationsData), qualificationsDataFilePath);
+            int recordsStaged = await context.CallActivityAsync<int>(nameof(QualificationsStager.InsertQualificationsDataIntoStaging), ofqualQualificationData);
+
+            return recordsStaged;
```

Comment:
Or this one

Reply:
Removed

Reply:
And there

Reply:
Sorry I must've missed something between all the stashing and staging. Now removed